<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabClearArea</class>
 <widget class="QWidget" name="TabClearArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>H:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="h_spinBox">
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="add_pushButton">
     <property name="text">
      <string>Add Entry</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="w_spinBox">
     <property name="maximum">
      <number>320</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="select_comboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select Entry:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="x_spinBox">
     <property name="maximum">
      <number>990</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="name_lineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="y_spinBox">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="color_pushButton">
     <property name="text">
      <string>Pick Color</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
